tern/formats/spdx/spdxtagvalue/generator.py

```diff
@@ -116,7 +116,7 @@ def get_layer_relationships(layer_obj, prev_layer_spdxref=None):
 
 def get_license_ref(pkg_license):
     '''Given one license, return a LicenseRef'''
-    return 'LicenseRef-' + pkg_license
+    return 'LicenseRef-' + pkg_license.replace(',', '-')
```

Comment:
How about using re.sub here for both `,` and `+` and for any other unacceptable characters we may have to deal with in the future? This way, we will not have to do it again in license block function and the generator function.